docs/src/main/sphinx/client/cli.rst

```diff
@@ -232,14 +232,18 @@ certificate usage:
     - The password for the keystore. This must match the password you specified
       when creating the keystore.
   * - ``--keystore-type``
-    - Keystore type.
+    - Determined by the keystore file format. The default keystore type is JKS.
+      This advanced option is only necessary for implementing a Java
```

Comment:
Lets make it the same in both section .. 

This advanced option is only necessary if you use a custom Java
Cryptography Architecture (JCA) provider implementation.